### DISCUSSION TITLE
BOM parts only from specific stock location

### DISCUSSION THREAD

**Discussion by SpArX0110** [inventree/InvenTree#4457] [Q&A]
Hello,

is there a possibility to have a BOM use only parts out of specific stock locations? So that the "can build" and the and "available parts" in the BOM only show the amount of parts in this specific stock.
If such a feature does not exist, could it be considered in a future release? Would be awsome for our stock management, which basically has different stock locations for different products.

**Comment by SchrodingersGat** [comment]
It is (probably) possible, but would require a bit of work to get all the elements in place. As this is not currently possible, and more of a feature request, I've raised a new issue: https://github.com/inventree/InvenTree/issues/4458

If this is something that your company wants attention given to promptly, maybe you'd be willing to sponsor development of this.
